django_webapps/scripts/db_population/parsers/common_parsers.py

```diff
@@ -58,13 +61,26 @@ def parse_timezone(tz_str: str) -> datetime.tzinfo | None:
     return None
 
 
+def find_any_email(email_str: str) -> str | None:
+    regex = re.compile(
+        r"([-!#-'*+/-9=?A-Z^-~]+(\.[-!#-'*+/-9=?A-Z^-~]+)*|\"([]!#-[^-~ \t]|(\\[\t -~]))+\")@([-!#-'*+/-9=?A-Z^-~]+(\.[-!#-'*+/-9=?A-Z^-~]+)*|\[[\t -Z^-~]*])"  # noqa: E501
+    )
+    match = re.search(regex, email_str)
+    if match is not None:
+        return match.group(0)
+    return None
+
+
 def parse_email(email_str: str) -> str | None:
-    email_str = email_str.strip()
+
+    email_str_result: str | None = email_str.strip().rstrip()
```

Comment:
`.strip().rstrip()` looks a bit odd (at least without a comment) :)

Reply:
It removes whitespaces from the right and left, yes, the API kind of strange )